conf/test_kraken2.config

```diff
@@ -0,0 +1,32 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running minimal tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines input files and everything required to run a fast and simple pipeline test.
+
+    Use as follows:
+        nextflow run nf-core/magmap -profile test,<docker/singularity> --outdir <OUTDIR>
+
+----------------------------------------------------------------------------------------
+*/
+
+process {
+    resourceLimits = [
+        cpus: 4,
+        memory: '15.GB',
+        time: '6.h'
+    ]
+}
+
+params {
+    config_profile_name        = 'Test profile'
+    config_profile_description = 'Minimal test dataset to check pipeline function'
+
+    // Input data
+    input           = params.pipelines_testdata_base_path + 'magmap/samplesheets/samplesheet.csv'
+    genomeinfo      = params.pipelines_testdata_base_path + 'magmap/testdata/genometest.csv'
+    sequence_filter = 'https://github.com/nf-core/test-datasets/raw/metatdenovo/test_data/rrna.fna.gz'
+    kraken2_db      = null
+    run_kraken2     = true
```

Comment:
I think Kraken should be default (with the `standard` database), so we should have a `--skip_kraken2` option rather than a `run_kraken2` param.